package.json

```diff
@@ -13,15 +13,15 @@
     "cytoscape-klay": "3.1.1",
     "cytoscape-popper": "^1.0.1",
     "lodash": "4.17.5",
-    "patternfly": "3.43.0",
-    "patternfly-react": "1.16.4",
+    "patternfly": "3.45.0",
+    "patternfly-react": "1.16.6",
```

Comment:
+1 
I'll need this one for the [Notification Drawer](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=NotificationDrawer&selectedStory=Notification%20Drawer&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

Reply:
Ok.

Reply:
Yep, actually that's the version we need for the NotificationDrawer, not the 1.16.6 :)